content/plugins/aws.md

```diff
@@ -163,12 +373,37 @@ However, sometimes, we map relationship behavior to plugin operations. The follo
 
+* `cloudify.aws.relationships.instance_contained_in_subnet` The `run_instances` operation looks for any relationships to a Subnet and creates the Instance in that Subnet. Otherwise, the instance is in the EC2 Classic VPC.
+
+* `cloudify.aws.relationships.instance_connected_to_load_balancer` This registers and EC2 instance with an Elastic Load Balancer.
+
+* `cloudify.aws.relationships.security_group_contained_in_vpc` A Security Group is created in EC2 classic unless it has this relationship. Then it will be created in the target VPC.
+
+* `cloudify.aws.relationships.volume_connected_to_instance` This attaches an EBS volume to an Instance.
+
+* `cloudify.aws.relationships.subnet_contained_in_vpc` This is required, so that when a Subnet is created, the plugin knows which VPC to create the Subnet in.
+
+* `cloudify.aws.relationships.routetable_contained_in_vpc` This is required, so that when a Route Table is created, the plugin knows which VPC to create the Route Table in. A Route Table can be created in only one VPC for its entire lifecycle.
+
+* `cloudify.aws.relationships.routetable_associated_with_subnet` A route table can be associated with no more than one subnet at a time.
+
+* `cloudify.aws.relationships.route_table_to_gateway` You can add multiple routes to route tables. You can add them as arguments to the create operation of the route table. For gateways, this is abstracted into a relationship. This adds a route to the source route table to the destination gateway. The gateway must have a CIDR block node property.
+
+* `cloudify.aws.relationships.gateway_connected_to_vpc` Attach either a VPN gateway or an Internet Gateway to a VPC.
+
+* `cloudify.aws.relationships.network_acl_contained_in_vpc` This is required for Network ACLs. A Network ACL must be contained in a VPC, otherwise the plugin does not know where to put it.
+
+* `cloudify.aws.relationships.network_acl_associated_with_subnet` This associates a Network ACL with a particular Subnet.
+
+* `cloudify.aws.relationships.route_table_of_source_vpc_connected_to_target_peer_vpc` This creates a VPC Peering Connection. A VPC Peering Connection is a connection between two VPCs. However, it requires a Route Table to associate the routes with. This will add routes to the source Route Table and to the Target VPC route table. You should also have a `cloudify.relationships.depends_on` relationship to the target VPC's route table, if you have a node_template to create one.
+
+* `cloudify.aws.relationships.dhcp_options_associated_with_vpc` Indicates with VPC to associate a DHCP options set with.
+
+* `cloudify.aws.relationships.customer_gateway_connected_to_vpn_gateway` Represents a VPC connection between a customer gateway and a VPN Gateway.
 
 
 # Types' Common Behaviors
```

Comment:
the default resource naming convention thing got stuff that use single quote instead of backticks